BankManagerConsoleApp/Model/Console/Menu.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  Console.swift
+//  BankManagerConsoleApp
+//
+//  Created by 우롱차, 민성 on 2022/04/28.
+//
+
+import Foundation
+
+struct Menu {
+    private enum MenuContstant {
+        static let menual =
+        """
+        1 : 은행개점
+        2 : 종료
+        """
+        static let InputMessage = "입력: "
+        static let openBank = "1"
+        static let finsh = "2"
```

Comment:
각 상수들이 어떤 값을 의미하는지 직관적으로 알 수 있는 네이밍이면 좋을 거 같아요.
ex. 내용(input) + 역할(Message)
```suggestion
        static let bankOpenInput = "1"
        static let finishInput = "2"
```

Reply:
네! 확실히 개굴이 쓴 네이밍이 좋네요!